packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -328,8 +328,8 @@ function apply_user_configs(configs)
     -- nixio.syslog("crit", "FBW apply_user_configs ssid "..ssid)
     uci_cursor:set("lime-defaults", 'wifi', 'ap_ssid', name)
     uci_cursor:set("lime-defaults", 'wifi', 'apname_ssid', name..'/%H')
-    uci_cursor:set("lime-defaults", 'wifi', 'adhoc_ssid', 'LiMe.'..name..'/%H')
-    uci_cursor:set("lime-defaults", 'wifi', 'ieee80211s_mesh_id', 'LiMe.'..name..'/%H')
+    uci_cursor:set("lime-defaults", 'wifi', 'adhoc_ssid', name..'.%H')
```

Comment:
I would not include the name of the community network there as it may create confusion in client devices

Reply:
I have seen the usefulness of having the name of the node at the time of scans and identify nodes that have the apname and apname closed. This is the case of nodes that for some reason does not link to the mesh (bad signal, configuration problem, etc). Or to identify if a node belongs to a community or another. Does LiMe.hostname make more sense than community.hostname?

Reply:
Yeah I think it would generate less confusion as it is more different to the ap(name)_ssid, take care that having different ssid works only for adhoc, while ieee80211s_mesh_id must be the same on all libremesh nodes, otherwise it won't connect